exercises/practice/lens-person/lens-person.spec.js

```diff
@@ -53,15 +51,8 @@ describe('bornAtLens', () => {
   });
 
   xtest('should ensure immutability by checking the original person object', () => {
-    expect(person).not.toEqual(
-      new Person(
-        person.name,
-        new Born(
-          new Address(2, 'Exercism street', 'Tamil Nadu', 'India'),
-          person.born.bornOn,
-        ),
-        person.address,
-      ),
+    expect(person.born.bornAt).toStrictEqual(
+      new Address(100, 'Hospital street', 'Tamil Nadu', 'India'),
     );
   });
```

Comment:
Same as above. Instead this should check that `person.address` is not changed when calling `streetLens.set`